src/rules/function-url-no-scheme-relative/__tests__/index.js

```diff
@@ -0,0 +1,81 @@
+import { testRule } from "../../../testUtils"
+import rules from "../../../rules"
+import { ruleName, messages } from ".."
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+
+  accept: [ {
+    code: "a { background: url(); }",
+  }, {
+    code: "a { background: url(''); }",
+  }, {
+    code: "a { background: url(\"\"); }",
+  }, {
+    code: "a { background: url(/); }",
+  }, {
+    code: "a { background: url(./); }",
+  }, {
+    code: "a { background: url(./file.jpg); }",
+  }, {
+    code: "a { background: url(../file.jpg); }",
+  }, {
+    code: "a { background: URL(../file.jpg); }",
+  }, {
+    code: "a { background: url('../file.jpg'); }",
+  }, {
+    code: "a { background: url(\"../file.jpg\"); }",
+  }, {
+    code: "a { background: url('/path/to/file.jpg'); }",
+  }, {
+    code: "a { background: url('https://www.google.com/file.jpg'); }",
+  }, {
+    code: "a { background: url('http://www.google.com/file.jpg'); }",
+  }, {
+    code: "a { background-image: url('data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs='); }",
+  }, {
+    code: "@font-face { font-family: 'foo'; src: url('/path/to/foo.ttf'); }",
+  }, {
+    code: "a { background: some-url(); }",
+    description: "ignore contain url function",
+  }, {
+    code: "a { background: url($image); }",
+    description: "ignore variable",
+  }, {
+    code: "a { background: url(@image); }",
+    description: "ignore variable",
+  }, {
+    code: "a { background: url(var(--image)); }",
+    description: "ignore variable",
+  } ],
+
+  reject: [ {
+    code: "a { background: url(//); }",
+    message: messages.rejected,
+    line: 1,
+    column: 5,
```

Comment:
Shouldn't these `column` numbers correspond to the beginning of the `url` argument? In this case, 21.